webview/src/plots/components/comparisonTable/ComparisonTable.test.tsx

```diff
@@ -535,14 +535,9 @@ describe('ComparisonTable', () => {
 
       const [headerWrapper] = getHeaders()
 
-      // eslint-disable-next-line testing-library/no-node-access
-      expect(headerWrapper.childElementCount).toBe(2)
       expect(
-        // eslint-disable-next-line testing-library/no-node-access
-        Object.values(headerWrapper.children)
-          .map(child => child.id)
-          .includes(endingNode.id)
-      ).toBe(true)
+        within(headerWrapper).getByTestId('comparison-drop-target')
+      ).toBeInTheDocument()
```

Comment:
[Q] How do these assertions differ/relate to one another?

Reply:
It's not wrapped the exact same way, but the test checks the same thing, it was just not written in a clear way before.